doc/src/fix_deposit.txt

```diff
@@ -57,7 +57,9 @@ keyword = {region} or {id} or {global} or {local} or {near} or {gaussian} or {at
     fix-ID = ID of "fix shake"_fix_shake.html command
   {units} value = {lattice} or {box}
     lattice = the geometry is defined in lattice units
-    box = the geometry is defined in simulation box units :pre
+    box = the geometry is defined in simulation box units 
```

Comment:
There has to be an explanation of this keyword and its values later in the doc page.  Put
the orientation option right after the mol and molfrac options since it pertains to molecules.
Explain what LAMMPS does when this is not specified (picks a random orientation).  This option
specifies a specific orientation for all desposited molecules?  I think you should explain
in words how to specify r[3] to get a desired orientation.  It's not obvious, since it involves
quaternion algebra.  What would the user specify to get no rotation?  I.e. just use the coords in the input
molecule file?  Also, I'm not clear if/how this should apply to 2d.  Shouldn't it be possible to randomize
or pick a specific orientation of a 2d moleclue for a 2d simulation?